gcc/rust/ast/rust-ast-full-test.cc

```diff
@@ -4025,14 +4027,15 @@ filename_from_path_attribute (std::vector<Attribute> &outer_attrs)
   return path_attr.get_attr_input ().as_string ();
 }
 
-std::string
+void
 Module::get_filename ()
```

Comment:
nit: I wonder if a better name could be used here, as `get_*` usually implies some kind of return.

Reply:
actually good point, What about changing this to module::process_file_path, of keep the name the same but actually return the string like before.

Reply:
You guys are completely right. I'll rename it when I get back home later

Reply:
Done in 13a9d29. Since we have to keep the string alive for a while, I think it's easier to keep it as a class member. Doing it this way avoids an extra std::move afterwards. But we can do that too

Reply:
good job!